@ecomplus/storefront-template/template/pages/@/meta.ejs

```diff
@@ -88,7 +88,16 @@ if (pathname) {
   if (_.widgets.length) {
     %>
     <script>
-      partytown = { forward: [] };
+      partytown = { 
+        resolveUrl: function (url, location) {
+          if (url.hostname !== 'analytics.google.com') {
```

Comment:
Mas de onde veio esse domínio aí ? 🤔 
Os collects to GA4 são em www.google-analytics.com

Reply:
```suggestion
          if (url.hostname !== 'www.google-analytics.com') {
```

Reply:
Se for o mesmo hostname pra GA4 e UA (pelo jeito é), deixamos a convenção de não usar partytown quem ainda estiver com UA (em breve ninguém mais usará)